pkgs/development/python-modules/arxiv2bib/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage, lib, fetchFromGitHub
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "arxiv2bib";
+  version = "1.0.8";
+
+  # Missing tests on Pypi
+  src = fetchFromGitHub {
+    owner = "nathangrigg";
+    repo = "arxiv2bib";
+    rev = version;
+    sha256 = "1kp2iyx20lpc9dv4qg5fgwf83a1wx6f7hj1ldqyncg0kn9xcrhbg";
+  };
+
+  # Required for tests only
+  checkInputs = [ mock ];
+
+  checkPhase = "python -m unittest discover -s tests";
```

Comment:
`${python.interpreter}` instead of `python`

Reply:
ehm python is undefined

Reply:
I add it to dependencies